src/MainWindow.vala

```diff
@@ -91,8 +91,6 @@ public class AppCenter.MainWindow : Gtk.ApplicationWindow {
 
         return_button.clicked.connect (view_return);
 
-        installed_view.get_apps.begin ();
```

Comment:
This is called in the constructor of the installed view, no need to call it again in this constructor.